add-rpms.sh

```diff
@@ -1,77 +1,73 @@
 #!/bin/bash
+source /usr/share/mixer-tools/helpers
```

Comment:
Note that you are installing this file to `$(datadir)/helpers`, so if mixer-tools is built with a datadir different than `/usr/share`, the file sourcing here will fail.

One solution would be to make the scripts templates instead, and substitute the correct values at configure time. The line in the template would look like `source @datadir@/mixer-tools/helpers`.